### PR TITLE
WINC-880: Update CCM jobs

### PR DESCRIPTION
**Pull request by aravindhp**
- Convert all vSphere and AWS CCM to weekly jobs
- Make Azure CCM jobs required as there is a binary packaged in WMCO  that we should be testing
- Add Slack reporting for CCM jobs

### CODE REVIEW

**Review by sebsoto — APPROVED** [review]

**Comment by elmiko** [comment]
@aravindhp :wave:

**Comment by mansikulkarni96** [comment]
Thanks fro working on this @aravindhp 
/lgtm

**Review by elmiko — COMMENTED** [review]
/lgtm

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ tests:
     from: src
   skip_if_only_changed: ^(?:docs|\.github)/|\.md$|^(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
 - as: aws-e2e-ccm-install
-  skip_if_only_changed: ^(?:docs|\.github)/|\.md$|^(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  cron: 0 9 * * 1
```
Comment (saifshaikh48) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:69]: I’m curious what time zone the job uses? Is this 9am UTC?

Comment (aravindhp) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:69]: Yes. Failures should show up on Monday morning. LMK if I miscalculated the conversion.

**Comment by aravindhp** [comment]
/pj-rehearse ack

Enough variants of the tests have passed across branches.

**Pull request merged**